### ISSUE TITLE
Improve Markdown support awareness for release notes

### ISSUE THREAD

**Issue by Dalmirog** [OctopusDeploy/Issues#3056]
As discussed in [our private chat](https://octopusdeploy.slack.com/archives/team-execution/p1483896430000022), the goal is to make it more obvious that we support Markdown for release notes. 

Few ideas:

A) Adding a help text on the UI next to the release notes editor saying "We support markdown" with a link to a site that shows you all the available MD formatting tricks.

B) Adding a brief *remember you can use markdown* on the help text of the `-ReleaseNotes` and `-ReleaseNotesFile` parameters of `Octo.exe`. `Octo.exe` users will be very bennefited from this as It is a very common practice to programatically generate the release notes from GH issues or TFS workitems, then dumping them on a file and use it with `-ReleaseNotesFile`.

C) Maybe mentioning this somewhere in our docs.

**Comment by fourpastmidnight** [comment]
This is already mentioned somewhere in your docs, as when I discovered this, I started using this feature. It would be helpful, however, to have some indicator near the Release Notes entry field to let a user know that Markdown is supported. Maybe it's enough to include a Markdown icon somewhere on the UI such that hovering over it shows a tooltip saying that Markdown is supported in that field?
